Polygon_mesh_processing/test/Polygon_mesh_processing/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ if (EIGEN3_FOUND)
   create_single_source_cgal_program("measures_test.cpp")
   create_single_source_cgal_program("triangulate_faces_test.cpp")
   create_single_source_cgal_program("test_pmp_remove_border_edge.cpp")
+  create_single_source_cgal_program("test.cpp")
```

Comment:
the name of the file is too generic and is actually not commited. I guess you did not want to commit that change.